README.md

```diff
@@ -243,7 +247,7 @@ puts "Please transfer fund to account no. #{@result.permata_va_number} in bank P
 See [our documentation](https://api-docs.midtrans.com/#charge-features) for other available options.
 
 
-## STEP 3: Receive notification callback
+## Receive notification callback
```

Comment:
Why the other section have numbering, but this one have no number? be consistent please